From 196c90d6876f2c6bba44fea7425dec4f6204ef37 Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Fri, 24 Apr 2020 14:48:42 +0900
Subject: [PATCH] extra config options 

enbale extra options

dns 
md5sum
mmc_rpmb and mmc_boot
usb keyboard
lzo zstd

---
 configs/khadas-edge-captain-rk3399_defconfig | 1 +
 configs/khadas-edge-rk3399_defconfig         | 1 +
 configs/khadas-edge-v-rk3399_defconfig       | 1 +
 configs/khadas-vim2_defconfig                | 1 +
 configs/khadas-vim3_defconfig                | 1 +
 configs/khadas-vim3l_defconfig               | 1 +
 configs/khadas-vim_defconfig                 | 1 +
 7 files changed, 7 insertions(+)

diff --git a/configs/khadas-edge-captain-rk3399_defconfig b/configs/khadas-edge-captain-rk3399_defconfig
index bb5e6e27..0bac3edb 100644
--- a/configs/khadas-edge-captain-rk3399_defconfig
+++ b/configs/khadas-edge-captain-rk3399_defconfig
@@ -39,2 +39,11 @@ CONFIG_MMC_SDHCI=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_DM_KEYBOARD=y
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/khadas-edge-rk3399_defconfig b/configs/khadas-edge-rk3399_defconfig
index 2963b30f..f3fe3373 100644
--- a/configs/khadas-edge-rk3399_defconfig
+++ b/configs/khadas-edge-rk3399_defconfig
@@ -37,2 +37,11 @@ CONFIG_MMC_SDHCI=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_DM_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_USB_KEYBOARD=y
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/khadas-edge-v-rk3399_defconfig b/configs/khadas-edge-v-rk3399_defconfig
index b0aef83c..37aa7d73 100644
--- a/configs/khadas-edge-v-rk3399_defconfig
+++ b/configs/khadas-edge-v-rk3399_defconfig
@@ -38,2 +38,11 @@ CONFIG_MMC_SDHCI=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_DM_KEYBOARD=y
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/khadas-vim2_defconfig b/configs/khadas-vim2_defconfig
index fb1f3a0c..8acd4009 100644
--- a/configs/khadas-vim2_defconfig
+++ b/configs/khadas-vim2_defconfig
@@ -43,2 +43,11 @@ CONFIG_DM_SPI_FLASH=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_DM_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/khadas-vim3_defconfig b/configs/khadas-vim3_defconfig
index e6eaaf02..5d1c9e5c 100644
--- a/configs/khadas-vim3_defconfig
+++ b/configs/khadas-vim3_defconfig
@@ -41,2 +41,11 @@ CONFIG_DM_SPI_FLASH=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_DM_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/khadas-vim3l_defconfig b/configs/khadas-vim3l_defconfig
index 532f765f..66f7ef57 100644
--- a/configs/khadas-vim3l_defconfig
+++ b/configs/khadas-vim3l_defconfig
@@ -41,2 +41,11 @@ CONFIG_DM_SPI_FLASH=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_DM_KEYBOARD=y
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
diff --git a/configs/khadas-vim_defconfig b/configs/khadas-vim_defconfig
index 0e788056..47ffe7ad 100644
--- a/configs/khadas-vim_defconfig
+++ b/configs/khadas-vim_defconfig
@@ -39,2 +39,11 @@ CONFIG_PHY_ADDR_ENABLE=y
 CONFIG_DM_ETH=y
+CONFIG_CMD_DNS=y
+CONFIG_LZO=y
+CONFIG_ZSTD=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+CONFIG_USB_KEYBOARD=y
+CONFIG_BOOTDELAY=1
+CONFIG_DM_KEYBOARD=y
+CONFIG_CMD_MD5SUM=y
 CONFIG_ETH_DESIGNWARE=y
-- 
2.17.1

